frappe_blurhash/events/on_file_upload.py

```diff
@@ -6,30 +6,45 @@
 
 
 def on_file_upload(doc, methods=None):
+    if not is_valid_file_for_blurhash_creation(doc):
+        return
```

Comment:
I think it'd be nice to log here as well, no? what do you think?

Reply:
log as in `error_log` ?

Reply:
we may spam the Error log doctype if say we only deal with invoices for example